DatePicker/DatePickerBase.js

```diff
@@ -210,15 +214,15 @@ const DatePickerBase = kind({
 		delete rest.rtl;
 
 		return (
-			<DateTime {...rest}>
+			<DateTime {...rest} className={classnames(className, dateTimeCss.timePicker)}>
```

Comment:
This in not right.
` className={classnames(className, dateTimeCss.timePicker)}` can be dropped entirely, and `timePicker` can simply be added to the styles className block above, if you really want both `datePicker` and `timePicker` classes applied, which also seems really weird...

```javascript
	styles: {
		css,
		className: 'datePicker timePicker'
	},
```
It's more likely that you'll just want one class on the className:
```javascript
	styles: {
		css,
		className: 'timePicker'
	},
```

Reply:
Thanks for suggestion. 
I left it as `<DateTime {...rest} className={css.datePicker}>` and added `datePicker` in styles className since this module refers to `DatePicker`. I hope this is what you expected.